lib/ddtrace/span.rb

```diff
@@ -217,5 +228,17 @@ def pretty_print(q)
         end
       end
     end
+
+    private
+
+    if defined?(JRUBY_VERSION)
+      def now_allocations
+        0
+      end
+    else
+      def now_allocations
+        GC.stat :total_allocated_objects
```

Comment:
I'm digging this feature! But just a couple of general questions:

1. How accurate/reliable is `GC.stat` for this? From what I remember playing around with this, it wasn't very deterministic. (Because other threads in the same process can allocate memory in parallel which changes this stat?) However, I can see this is exactly what the Rails guys did, so I presume they did their homework on this. I suppose as long as its in the ballpark and its generally useful even if not perfectly accurate, I'd be willing to give it the benefit of the doubt.
2. Does this work for Ruby 1.9.3+? (JRuby excepted?) If not we might need to add in some compatibility or short circuit for it.

Reply:
1. It seems to be good enough to use in Rails, and it's it doesn't have to be 100% precise to be useful in practice, especially if you aggregate.
2. I've added a compatibility check.